javaparser-core/src/main/java/com/github/javaparser/ast/stmt/CatchClause.java

```diff
@@ -68,6 +69,7 @@ public CatchClause(final Range range,
 		v.visit(this, arg);
 	}
 
+    // TODO Should we deprecate those ?
 	public BlockStmt getCatchBlock() {
```

Comment:
I prefer to keep them deprecated untill we reach like another major version, I don't like when methods just doesn't exist anymore without providing information, this way we let them fix it on the long run.